generate-nearby.js

```diff
@@ -1,41 +1,45 @@
-let faker = require('faker');
-let fs = require('fs');
-let data = [];
+const faker = require('faker');
+const fs = require('fs');
+
 const total = 10000000;
 console.log('starting');
-let generate = (writer, encoding, callback) => {
+const generate = (writer, encoding, callback) => {
   let i = total;
-  let write = () => {
+  const write = () => {
     let ok = true;
     do {
-      let data = {
+      const data = {
         name: faker.address.city() + faker.address.streetName(),
         place_id: i,
-        google_rating: (Math.random() * 4 + 1).toFixed(1),
-        zagat_rating: (Math.random() * 4 + 1).toFixed(1),
-        photos: [Math.floor(Math.random() * 100), Math.floor(Math.random() * 100), Math.floor(Math.random() * 100), Math.floor(Math.random() * 100)],
-        neighborhood: faker.address.streetName() + faker.address.citySuffix(), 
-        price_level: Math.floor((Math.random() * 4 + 1)),
+        google_rating: ((Math.random() * 4) + 1).toFixed(1),
+        zagat_rating: ((Math.random() * 4) + 1).toFixed(1),
+        photos: [Math.floor(Math.random() * 100),
+          Math.floor(Math.random() * 100), Math.floor(Math.random() * 100),
+          Math.floor(Math.random() * 100)],
+        neighborhood: faker.address.streetName() + faker.address.citySuffix(),
+        price_level: Math.floor(((Math.random() * 4) + 1)),
         types: faker.lorem.word(),
-        nearby: [Math.floor(Math.random() * total), Math.floor(Math.random() * total), Math.floor(Math.random() * total),
-          Math.floor(Math.random() * total), Math.floor(Math.random() * total), Math.floor(Math.random() * total)]
+        nearby: [Math.floor(Math.random() * total),
+          Math.floor(Math.random() * total), Math.floor(Math.random() * total),
+          Math.floor(Math.random() * total), Math.floor(Math.random() * total),
+          Math.floor(Math.random() * total)],
       };
-      i--;
+      i -= 1;
       if (i === 0) { // end
-        writer.write(JSON.stringify(data) + ']', encoding, callback);
+        writer.write(`${JSON.stringify(data)}]`, encoding, callback);
```

Comment:
The template literal to insert the commas and closing bracket seems kinda of weird.